test/e2e/framework/fixture.go

```diff
@@ -175,6 +179,11 @@ func NewOrganizationFixture(t *testing.T, server RunningServer) (orgClusterName
 }
 
 func NewWorkspaceFixture(t *testing.T, server RunningServer, orgClusterName string, workspaceType string) (clusterName string) {
+	schedulable := workspaceType == "Universal"
+	return NewWorkspaceWithScheduling(t, server, orgClusterName, workspaceType, schedulable)
+}
+
+func NewWorkspaceWithScheduling(t *testing.T, server RunningServer, orgClusterName string, workspaceType string, schedulable bool) (clusterName string) {
```

Comment:
NewWorkspaceWithWorkloads.

There is many more kinds of scheduling.

Reply:
Removed.